cartography/intel/azure/tenant.py

```diff
@@ -0,0 +1,49 @@
+import logging
+from typing import Dict
+
+import neo4j
+
+from .util.credentials import Credentials
+from cartography.util import run_cleanup_job
+from cartography.util import timeit
+
+logger = logging.getLogger(__name__)
+
+
+def get_tenant_id(credentials: Credentials) -> str:
+    return credentials.get_tenant_id()
+
+
+def load_azure_tenant(neo4j_session: neo4j.Session, tenant_id: str, current_user: str, update_tag: int) -> None:
+    query = """
+    MERGE (at:AzureTenant{id: {TENANT_ID}})
+    ON CREATE SET at.firstseen = timestamp()
+    SET at.lastupdated = {update_tag}
+    WITH at
+    MERGE (ap:AzurePrincipal{email: {CURRENT_USER}})
```

Comment:
Do
```
MERGE (ap:AzurePrincipal{id: {CURRENT_USER}})
ON CREATE SET ap.email = {CURRENT_USER},
ap.lastupdated ...
```

Add an index for `id`. You can keep the existing index on `AzurePrincipal(email)`.

Reply:
neat. I have updated code to add this logic.